indico/modules/events/abstracts/operations.py

```diff
@@ -211,7 +211,7 @@ def judge_abstract(abstract, abstract_data, judgment, judge, contrib_session=Non
     if judgment == AbstractAction.accept:
         abstract.state = AbstractState.accepted
         abstract.accepted_track = abstract_data.get('accepted_track')
-        if abstract_data.get('override_contrib_type'):
+        if abstract_data.get('override_contrib_type') or abstract_data.get('accepted_contrib_type'):
```

Comment:
Maybe it would be better if you swap these two checks, I think it's more frequent that we'll have an accepted_contrib_type than the override_contrib_type.